.changeset/plain-mails-tan.md

```diff
@@ -0,0 +1,24 @@
+---
+"@fluidframework/aqueduct": minor
+"@fluidframework/container-runtime": minor
+---
+---
+"section": deprecation
+---
+
+The ContainerRuntime class is now deprecated
+
+The class `ContainerRuntime` is deprecated and will no longer be exported starting in version 2.20.0.
+
+There are two possible migration paths to stop using `ContainerRuntime`:
+
+* When using it as a type, replace it with an interface like `IContainerRuntime` (or in rare cases, `IRuntime`)
```

Comment:
Do we have more details on which scenarios would lead to one or the other? If so, would be nice to include here.

Reply:
Probably could just leave off the `IRuntime` bit.  I think that's only exposed because of `instantiateRuntime` and honestly isn't applicable to any time an app would really have this thing.

Reply:
```suggestion
* When using it as a type, replace it with an interface like `IContainerRuntime`
```